traffic/data/adsb/opensky_impala.py

```diff
@@ -838,6 +842,10 @@ def history(
             )
 
         elif departure_airport is not None:
+            if airport is not None:
+                raise RuntimeError(
+                    "airport may not be set if " "departure_airport is set"
+                )
```

Comment:
I understand that `arrival_airport`/`departure_airport` and `airport` cannot coexist, so I added the relevant error